### ISSUE TITLE
Not working on Windows 10 14931+ due to hotpatch changes

### ISSUE THREAD

**Issue by Doonut** [riverar/uxstyle#37]
UXStyle is not working on this build. It is a Fast Ring update, But Id love to be able to theme on this build. The Personalize menu crashes as soon as you open it.

**Comment by DALIGR19** [comment]
Yeah, same thing for 14936 and 14942 ! Microsoft changed something i guess.

**Comment by ghost** [comment]
Same issue on the newer builds.

**Comment by Mattabase** [comment]
Same Issue while using build 14986

**Comment by shinji257** [comment]
This bug has gotten some attention on the Windows Insiders subreddit.  Don't know if the author is interested in investigating/resolving the issue or not.  It has been a while since an update has come around.  Probably changes in the theme engine that we may or may not of noticed.

https://www.reddit.com/r/windowsinsiders/comments/5ngnlf/psa_uxstyle_custom_theme_patcher_is_broken_as_of/

**Comment by riverar** [comment]
Will investigate.

**Comment by riverar** [comment]
Hotpatch bytes are disappearing in newer builds of Windows 10. The disassembler will need some small tweaks to support these newer builds.

**Comment by falconwingz** [comment]
I hope its fixed soon. because its almost a year already.

**Comment by ghost** [comment]
RIP?

**Comment by DonSYS91** [comment]
I guess RIP

**Comment by riverar** [comment]
The code is still open source, UxStyle is very low priority due to its low usage.

**Comment by ghost** [comment]
I'd use it in a heartbeat over dirty patching system files that get modified in updates, but there's no choice right now

**Comment by MrGRiM01** [comment]
Same here mate

**Comment by timaxapa** [comment]
No C++ coders here?
